frontends/concrete-python/examples/tfhers/example.py

```diff
@@ -0,0 +1,175 @@
+import os
+from functools import partial
+
+import click
+import numpy as np
+
+from concrete import fhe
+from concrete.fhe import tfhers
+
+########## Params #####################
+LWE_DIM = 909
+GLWE_DIM = 1
+POLY_SIZE = 4096
+PBS_BASE_LOG = 15
+PBS_LEVEL = 2
+MSG_WIDTH = 2
+CARRY_WIDTH = 3
+ENCRYPTION_KEY_CHOICE = tfhers.EncryptionKeyChoice.BIG
+LWE_NOISE_DISTR = 0
+GLWE_NOISE_DISTR = 2.168404344971009e-19
+#######################################
+
+assert GLWE_DIM == 1, "glwe dim must be 1"
```

Comment:
why?

Reply:
if it's other than 1, then how would we convert our LWE secret key into a GLWE one in TFHE-rs?

Reply:
not sure about that... so ok for now